.circleci/config.yml

```diff
@@ -28,8 +28,8 @@ jobs:
             yarn
             yarn dist
             cd applications/desktop/dist/
-            chmod a+x nteract-0.11.6-x86_64.AppImage
-            ./nteract-0.11.6-x86_64.AppImage --appimage-extract
+            chmod a+x nteract-0.11.7-x86_64.AppImage
```

Comment:
Since we know we're building the dist here, we could instead run `chmod a+x nteract*.AppImage`

Reply:
That's a good idea, then we don't have to do this every time 👍